server/mediapublic/views.py

```diff
@@ -130,6 +130,16 @@ class UsersResource(ResourceMixin):
     cls = Users
 
 
+@resource(collection_path='/social_medias', path='/social_medias/{id}',
```

Comment:
So I added a table and endpoint for "Social Media".  You pass it a provider and a username, and they can be anything.  so "Titter" and "@arbiterofbits", rather than having to add more things to the table each time a new social media thing comes out.